config/initializers/bulkrax.rb

```diff
@@ -25,96 +25,91 @@
-      "types" => { from: ["type"], split: /\s*[;]\s*/, parsed: true },
-      "remote_files" => { from: ['thumbnail_url'], parsed: true },
-      'parents' => { from: ['parents'], related_parents_field_mapping: true },
-      'children' => { from: ['children'], related_children_field_mapping: true }
-    },
-    "Bulkrax::CdriParser" => {
-      "contributing_institution" => {from: ['publisher'] },
-      "creator" => { from: ["creator"], split: /\s*[;]\s*/ },
-      "date" => { from: ['date', 'pub_date'], split: /\s*[;]\s*/ },
-      "language" => { from: ["language"], parsed: true, split: /\s*,\s*/ },
-      "subject" => { from: ["subject"], parsed: true, split: /\s*[;]\s*/ }
-    },
-    "Bulkrax::OaiIaParser" => {},
-    "Bulkrax::OaiSetsParser" => {},
-    "Bulkrax::OaiOmekaParser" => {},
-    "Bulkrax::OaiPtcParser" => {},
-    "Bulkrax::CsvParser" => {},
-    "Bulkrax::OaiQualifiedDcParser" => {}
+  config.field_mappings['Bulkrax::CdriParser'] = {
+    'contributing_institution' => {from: ['publisher'] },
+    'creator' => { from: ['creator'], split: /\s*[;]\s*/ },
+    'date' => { from: ['date', 'pub_date'], split: /\s*[;]\s*/ },
+    'language' => { from: ['language'], parsed: true, split: /\s*,\s*/ },
+    'subject' => { from: ['subject'], parsed: true, split: /\s*[;]\s*/ }
   }
 
-  # omeka - uses the same mappings as oai_dc
-  config.field_mappings["Bulkrax::OaiDcParser"].each {|key,value| config.field_mappings["Bulkrax::OaiOmekaParser"][key] = value }
+  config.field_mappings['Bulkrax::CsvParser'] = default_field_mapping.merge({
+    'abstract' => { from: ['abstract'], excluded: true },
+    'alternative_title' => { from: ['alternative_title'], split: /\s*[;]\s*/ },
+    'children' => { from: ['children'], related_children_field_mapping: true },
+    'contributing_institution' => { from: ['contributing_institution'], split: /\s*[;]\s*/ },
+    'extent' => { from: ['extent'], split: /\s*[;]\s*/ },
+    'format_digital' => { from: ['format', 'format_digital'], parsed: true, split: /\s*[;]\s*/ },
+    'format_original' => { from: ['format_original'], split: /\s*[;]\s*/, parsed: true },
+    'has_manifest' => { from: ['has_manifest'] },
+    'parents' => { from: ['parents'], related_parents_field_mapping: true },
+    'place' => { from: ['place'], split: /\s*[;]\s*/ },
+    'remote_files' => { from: ['thumbnail_url', 'remote_files'], parsed: true },
+    'remote_manifest_url' => { from: ['remote_manifest_url'], split: /\s*[;]\s*/ },
+    'rights_holder' => { from: ['rights_holder'], split: /\s*[;]\s*/ },
+    'rights_statement' => { from: ['rights', 'rights_statement'], split: /\s*[;]\s*/ },
+    'time_period' => { from: ['time_period'], split: /\s*[;]\s*/ },
+    'types' => { from: ['type', 'types', 'resource_type'], split: /\s*[;]\s*/, parsed: true }
+  })
+
+  config.field_mappings['Bulkrax::OaiDcParser'] = default_field_mapping.merge({
+    # add, remove or override custom mappings for this parser here
+  })
```

Comment:
Can we leave these commented out until they're needed? 

```suggestion
  # config.field_mappings['Bulkrax::OaiDcParser'] = default_field_mapping.merge({
    # add, remove or override custom mappings for this parser here
  # })
```

Reply:
you mean until the parser is needed? this parser, and the ones in the comments below, are in use already, they just don't have any additional mappings outside of the `default_field_mapping`. I wrote the line with the `merge` and comment as a hint to future devs. instead of just doing `config.field_mappings['Bulkrax::OaiDcParser'] = default_field_mapping`.

previously, this file listed everything I have inside of `default_field_mapping`, inside the `Bulkrax::OaiDcParser` mapping. then it copied that mapping into every other parser. and added new or overridden mappings by constantly updating the config on every line instead of as a single hash.

Reply:
Ahh, no you're 100% correct. They have to be defined still. Disregard this suggestion